containerengine/hello-world/README.md

```diff
@@ -1,29 +0,0 @@
-<img src="https://avatars2.githubusercontent.com/u/2810941?v=3&s=96" alt="Google Cloud Platform logo" title="Google Cloud Platform" align="right" height="96" width="96"/>
-
-# Google Container Engine World sample
-
-This is a basic hello world sample.
-
-View the [documentation][docs] or the [source code][code].
-
-[docs]: https://cloud.google.com/container-engine/docs/quickstart
```

Comment:
This [quickstart](https://cloud.google.com/container-engine/docs/quickstart) uses `us-docker.pkg.dev/google-samples/containers/gke/hello-app:1.0` — which is hosted and maintained at [github.com/GoogleCloudPlatform/kubernetes-engine-samples/tree/main/hello-app](https://github.com/GoogleCloudPlatform/kubernetes-engine-samples/tree/main/hello-app).